src/engine/renderer/glsl_source/lighttile_fp.glsl

```diff
@@ -123,8 +123,8 @@ void main() {
 	only process 1 / 4 of different lights for each layer, extra lights going into the last layer. This can fail to add some lights
 	if 1 / 4 of all lights is more than the amount of lights that each layer can hold (16). To fix this, we'd need to either do this on CPU
 	or use compute shaders with atomics so we can have a variable amount of lights for each tile. */
-	for( uint i = u_lightLayer; i < u_numLights; i += 4 ) {
```

Comment:
The prior version seems more readable to me.

Reply:
Also the proposed version is going to have an off by 1 bug if the number of lights is not divisible by 4.

Reply:
Hmm, true.

Reply:
Fixed now.